File: app/Flecha Digital/Model/TipoBoleto.cs
using System;
using System.Collections;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Flecha_Digital.Model
{
    public class TipoBoleto
    {
        public int cve_tipo { get; set; }
        public string descripcion { get; set; }
        public int disponibles { get; set; }
    }
}

File: server/services/viajes.api/models/classes/tipoBoleto.js
class tipoBoleto{
    //Clave tipo: se obtiene 
    //Descripción: se obtiene
    //Disponibles: se obtiene
    constructor(cve_tipo, descripcion, disponibles){
        this.cve_tipo = cve_tipo || null;
        this.descripcion = descripcion || null;
        this.disponibles = disponibles || null;
    }
}

module.exports = tipoBoleto;